### PR TITLE
Hotfix/manifestos

### PR DESCRIPTION
**Pull request by VirginiaDooley**
This work includes various manifesto enhancements and person page fixes. Extracting the manifestos into it's own card reduces clutter on the policy card. 

Screen Shot 2021-03-30 at 12 43 32 PM

### CODE REVIEW

**Comment by pmk01** [comment]
One thing I should have noticed earlier: can we remove 'UK'?

File: wcivf/apps/elections/tests/test_models.py [hunk 1]
```diff
@@ -158,6 +158,30 @@ def test_friendly_name_mayoral(self, post_election):
...
+        future = PostElectionFactory(
+            ballot_paper_id="parl.cities-of-london-and-westminster.2021-05-06",
+            post=post,
+            election=ElectionFactoryLazySlug(
+                election_date="2021-5-6", current=True, election_type="parl"
+            ),
+        )
+
+        assert oldest.past_registration_deadline is True
+        assert future.past_registration_deadline is False
```
Comment (michaeljcollinsuk) [inline wcivf/apps/elections/tests/test_models.py:183]: When we get past the registration deadline for this election (Is it 19th April?) this test will fail. We can get around this by using Freezegun (docs for reference https://pypi.org/project/pytest-freezegun/) which we already have installed and are using in other tests, to fix the date at the time the test runs to before the deadline.

File: wcivf/apps/elections/tests/test_models.py [hunk 2]
```diff
@@ -158,6 +158,30 @@ def test_friendly_name_mayoral(self, post_election):
             == "Greater London Authority mayoral election"
         )
 
+    @pytest.mark.django_db
+    def test_past_registration_deadline(self, post_election):
+        # election = ElectionWithPostFactory()
```
Comment (michaeljcollinsuk) [inline wcivf/apps/elections/tests/test_models.py:163]: Can the commented out lines be deleted?

File: wcivf/apps/elections/tests/test_models.py (hunk 1 above)
Comment (michaeljcollinsuk) [reply wcivf/apps/elections/tests/test_models.py:183]: Example of other tests where we do this https://github.com/DemocracyClub/WhoCanIVoteFor/blob/64347d36f05df59fd9808eeb885932b4994a1912/wcivf/apps/parishes/tests/test_models.py#L10

**Pull request merged**